components/bl-popover-component/src/helpers.js

```diff
@@ -0,0 +1,46 @@
+export const translatePopover = (buttonElement, root, position) => {
```

Comment:
Do you write all that logic by yourself? If no, could you share an reference?

Reply:
I wrote it myself